OpenTabletDriver.Web/Views/Wiki/FAQ/Linux.md

```diff
@@ -0,0 +1,241 @@
+
+For sanity reasons, make sure that your tablet is actually plugged in to your computer, and is detected by your OS.
+
+You can check this with `lsusb` or a GUI utility. `dmesg` may also be handy to inspect USB plug or unplug events
+
+Once you are 100% sure your tablet is connected to your computer, and your kernel is seeing it,
+ensure that your specific tablet make and model is supported by checking our list of supported tablets [here](/Tablets).
+
+If you plugged in your tablet _before_ installing OpenTabletDriver, please replug your tablet, otherwise detection might be impaired.
+
+We currently do not support tablets that connect directly to a computer via Bluetooth,
+and there is only limited support for tablets that use wireless USB receivers, so please make sure your tablet is connected via cable.
+
+If you installed OpenTabletDriver with a package manager, and replugging the tablet does not help, you might need to reboot your computer.
+
+Please reboot before attempting the solutions below to avoid misconfiguration.
+
+#### It is a supported tablet, but still doesn't detect after a reboot?
+
+Look for a log that resembles the following:
+
+<small class="text-muted">Hint: You can see the console log under the "Console" tab in OpenTabletDriver.UX. You will better find problems when you
+Switch from `Info` to `Debug` mode in the bottom left of the Console panel.
+</small>
+
+##### hid_uclogic kernel module interference {#hid_uclogic}
+
+> Another tablet driver found: UC Logic\
+> ArgumentOutOfRangeException Value range is [0, 15]. (Parameter 'value')
+
+These errors may happen if the `hid_uclogic` kernel module is loaded at the same time as OpenTabletDriver.
+
+You can remove and blacklist this module to stop it from loading in the future by running the following commands:
+```bash
+echo "blacklist hid_uclogic" | sudo tee -a /etc/modprobe.d/blacklist.conf
+sudo rmmod hid_uclogic
+```
+
+##### Missing udev rules {#udev}
+
+> Not permitted to open HID class device at /dev/hidrawX
+
+If you have followed the previous instructions, usually this is the only reason OpenTabletDriver will fail to detect your tablet when it's supported.
+
+The issue is resolved by making sure that you have proper udev rules set up.
+
+###### From Package Manager
+
+If you installed OpenTabletDriver via your package manager, you may just need to reload and trigger the rules:
+```bash
+sudo udevadm control --reload-rules
+sudo udevadm trigger
+```
+
+If it still doesn't work after this point, and you installed OpenTabletDriver with a package manager, please reboot, some systems may require this.
+
+###### Portable install
+
+If you installed OpenTabletDriver without using a package manager you will not have proper udev rules set up and must build your own.
+It is highly recommended that you use your package manager where available.
+
+This process will require you to have the dotnet 7 SDK installed before proceeding:
```

Comment:
I would hyperlink here to where the dotnet 7 sdk can be downloaded

Reply:
This is distribution dependent, is it not? But if a link is needed, I suggest https://github.com/dotnet/core/blob/main/linux.md or better.

The "Install .NET on Linux instuctions" provided on the download page aren't great IMO, but it does link to the above page which I think is more safe, since it seems to prefer repo packages.

Reply:
If anything we should refer the user to their package manager, on Linux you should never really manage this yourself.